docs/index.yml

```
@@ -28,25 +28,138 @@ highlightedContent:
+    - title: "Create your first web app."
       itemType: learn
       url: https://dotnet.microsoft.com/learn/aspnet/hello-world-tutorial/intro
 
-# productDirectory section (optional)
-productDirectory:
+# conceptualContent section (optional)
+conceptualContent:
+# itemType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | tutorial | video | whats-new
+  title: ".NET: Free. Cross-platform. Open source." # < 60 chars (optional)
+  summary: "A developer platform for building all your apps: web, mobile, desktop, gaming, IoT, and more. Supported on Windows, Linux, and macOS."
+  items:
+    # Card
+    - title: .NET Core
+      links:
+        - url: core/get-started.md
+          itemType: get-started
+          text: Get started
+        - url: core/whats-new/dotnet-core-3-0
+          itemType: whats-new
+          text: What's new in .NET Core 3.0
+        - url: core/deploying/index.md
+          itemType: deploy
+          text: Deploying .NET Core apps
+        - url: core/docker/build-container.md
+          itemType: tutorial
+          text: Containerize .NET Core apps with Docker
+        - url: core/tutorials/index.md
+          itemType: tutorial
+          text: Tutorials
+        - url: core/porting/index.md
+          itemType: how-to-guide
+          text: Port  apps from .NET Framework
+      # footerLink (optional)
+      footerLink:
+        url: core/index.md
+        text: See more
+    # Card
+    - title: .NET Concepts
+      links:
+        - url: https://channel9.msdn.com/Series/NET-Core-101/What-is-NET
+          itemType: video
+          text: What is .NET?
+        - url: standard/get-started.md
+          itemType: get-started
+          text: Get started
+        - url: standard/tour.md
+          itemType: overview
+          text: Tour of .NET
+        - url: standard/components.md
+          itemType: concept
+          text: .NET architectural components
+        - url: standard/net-standard/index.md
+          itemType: overview
+          text: .NET Standard
+        - url: framework/index.md
+          itemType: overview
+          text: .NET Framework (Windows) apps
+      # footerLink (optional)
+      footerLink:
+        url: standard/index.md
+        text: See more
+    # Card
+    - title: Develop .NET apps
+      links:
+        - url: core/sdk/index.md
+          itemType: download
+          text: Download the .NET Core SDK
+        - url: dotnet/core/tools/index.md
+          itemType: overview
+          text: .NET Core CLI overview
+        - url: core/testing/index.md
+          itemType: tutorial
+          text: Unit testing with .NET Core apps
+        - url: core/porting/index.md
+          itemType: how-to-guide
+          text: Porting from .NET Framework to .NET Core
+        - url: desktop-wpf/overview/index.md
+          itemType: overview
+          text: Create .NET Core desktop apps
```

Should Windows be mentioned somehow? For example:
```suggestion
          text: Create .NET Core desktop apps for Windows
```